include/alpaka/rand/TinyMT/LICENSE.txt

```diff
@@ -0,0 +1,30 @@
+Copyright (c) 2011, 2013 Mutsuo Saito, Makoto Matsumoto,
```

Comment:
This license seems to be compatible with ours.
@ComputationalRadiationPhysics/alpaka-maintainers Does anyone see any problems.

Reply:
Yep, it's "new", "revised" or ["modified" 3-clause BSD](https://en.wikipedia.org/wiki/BSD_licenses#3-clause_license_(%22BSD_License_2.0%22,_%22Revised_BSD_License%22,_%22New_BSD_License%22,_or_%22Modified_BSD_License%22)) which is GPL compatible. We have to ship the license (added) and state it, maybe more centrally accessible, was well.